test/setup/main.tf

```diff
@@ -18,11 +18,11 @@ module "project" {
   source  = "terraform-google-modules/project-factory/google"
   version = "~> 8.0"
 
-  name              = "ci-group"
-  random_project_id = "true"
-  org_id            = var.org_id
-  folder_id         = var.folder_id
-  billing_account   = var.billing_account
+  name                 = "ci-group"
```

Comment:
You can add module here for CI triggers
https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/terraform/test-org/org/locals.tf

Reply:
will do. Testing might be tricky for this module as granting Google Workspace Group Admin role to a SA seems manual and can't be done via TF: https://cloud.google.com/identity/docs/how-to/setup#assigning_an_admin_role_to_the_service_account

Reply:
We could take the same approach we did for the Project Factory module of predefining an SA which gets reused for each test run: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/terraform/test-org/org/gsuite.tf

Reply:
Added tests and CI is working.